Dockerfile

```diff
@@ -18,3 +18,18 @@ COPY scripts/pre-stop.sh /usr/local/lib/synthetic-monitoring-agent/pre-stop.sh
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
 ENTRYPOINT ["/usr/local/bin/synthetic-monitoring-agent"]
+
+# third stage with alpine base for better access to chromium
+FROM alpine:3.18 as with-browser
+
+RUN apk --no-cache add chromium-swiftshader
+
+ENV SM_CHROME_BIN=/usr/bin/chromium-browser
+ENV SM_CHROME_PATH=/usr/lib/chromium/
```

Comment:
What are these two for?

Reply:
I wondered if providing an env var might be useful instead of relying on PATH to have `chromium`/assuming the binary name, for when sm-agent leverages it.

Reply:
Will we add `xk6-browser` to the k6 setup for this? We may want to add `K6_BROWSER_HEADLESS` and `K6_BROWSER_ARGS` for that

Reply:
`xk6-browser` is now part of `k6`, so it's already there :D

I believe it should already be looking for the correct path, but we definitely want to tweak `K6_BROWSER_ARGS`. For instance, `--disable-dev-shm-usage` is required for some webpages to not crash on a container.

Reply:
The K6 Dockerfile seems to do the following:
```
# no-sandbox chrome arg is required to run chrome browser in
# alpine and avoids the usage of SYS_ADMIN Docker capability
ENV K6_BROWSER_ARGS=no-sandbox
```

Should we be using no-sandbox as well? I'll add it tentatively to this PR, will re-request your review again before I merge anything.